oci/oci.go

```diff
@@ -17,8 +17,8 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/kubernetes-incubator/cri-o/utils"
 	"github.com/containernetworking/cni/pkg/ns"
+	"github.com/kubernetes-incubator/cri-o/utils"
```

Comment:
@runcom yeah, I mean you are right in this PR, this is a gofmt problem brought before.